tests/parameters_test_input_files.ini

```diff
@@ -1,7 +1,7 @@
 [GENERAL]
 ###########################################################################
 # Number of simulation snapshots
-num_snapshots = 11
+num_snapshots = 1
```

Comment:
Should change it back to 10.

Reply:
Should it be 10 or 100?

Reply:
Changed to 10 by commit 04d6f5de34a7d9a8471c6ed1ce20bae29a3af38d